charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/templates/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Need a PSP if we're planning to support deploying this chart in hardened clusters or clusters that impose CIS scans, see https://confluence.suse.com/display/EN/Feature+Charts+Helm+Requirements#FeatureChartsHelmRequirements-AddingHardenedClusterSupport for more information.

Feel free to comment if that's not in scope for this PR 🙂

Reply:
same comment for CSI driver daemonset